### ISSUE TITLE
files missed in move to src/main/java shuffle?

### ISSUE THREAD

**Issue by markusicu** [unicode-org/unicodetools#163]
Most of the unicodetools Java code is now in https://github.com/unicode-org/unicodetools/tree/main/unicodetools/src/main/java/org/unicode
and in https://github.com/unicode-org/unicodetools/tree/main/unicodetools/src/test/java/org/unicode
but there are two files left in https://github.com/unicode-org/unicodetools/tree/main/unicodetools/org/unicode
(note the missing src/main/java | src/test/java infix)
and Eclipse seems to ignore them for things like finding all call sites for Settings.DataDir. (This is why I added a comment there saying it's unused.)

@srl295 Is this just a matter of moving these two files into the regular main|test hierarchies? Or is there more to it?

**Comment by srl295** [comment]
You can just move them. What concerns me is this may have been a merge
conflict.

**Comment by srl295** [comment]
That is to say: perhaps the file changed after I made my change but before
the PR was merged. Check that files history.

**Comment by srl295** [comment]
Yeah that’s exactly what happened. See
https://github.com/unicode-org/unicodetools/commits/main/unicodetools/org/unicode/tools/UpdateJspFiles.java

**Comment by markusicu** [comment]
Ok, thanks. I moved these files, but one then doesn't compile:
```
	renamed:    unicodetools/org/unicode/tools/UpdateJspFiles.java -> unicodetools/src/main/java/org/unicode/tools/UpdateJspFiles.java
	renamed:    unicodetools/org/unicode/propstest/CheckEmojiProps2.java -> unicodetools/src/test/java/org/unicode/propstest/CheckEmojiProps2.java
```
org.unicode.tools.UpdateJspFiles.java contains the following lines that Eclipse cannot resolve:
```
import org.unicode.propstest.CopyPropsToUnicodeJsp;
import org.unicode.propstest.ListProps;
```
I suspect that that's because main/... library code does not want to depend on test/... code. (Which makes sense.)

@srl295 @macchiati Is it ok if I move CopyPropsToUnicodeJsp and ListProps into the library code? And whatever other test classes they might depend on?

**Comment by srl295** [comment]
@markusicu  sounds like the right path.  Of course, previously we didn't have this test/library distinction. But CopyPropsToUnicodeJsp is a tool and not a test.

**Comment by srl295** [comment]
these files may have been actually _created_ after the Maven PR was opened, and I just missed them (besides the JSP tests, which were intentionally missed for that phase, #146 ).  If they were actual merge conflicts I would have had to sort them out in the PR.

**Comment by markusicu** [comment]
Fix in #166
